ix-dev/community/mariadb/ix_values.yaml

```
@@ -0,0 +1,13 @@
+images:
+  image:
+    repository: mariadb
+    tag: 11.7
+  v10_image:
+    repository: mariadb
+    tag: 10
```

```suggestion
    tag: 10.11
```
Lets pin it, and the bot will open PRs for newer versions, so we can do some "smoke test" on ci. :)